server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -493,14 +505,10 @@ private void switchToTransportProtocol(Channel channel) {
         addTransportHandler.accept(pipeline);
     }
 
-    private void sendParams(Channel channel, CoordinatorSessionSettings sessionSettings) {
-        Messages.sendParameterStatus(channel, "crate_version", Version.CURRENT.externalNumber());
-        Messages.sendParameterStatus(channel, "server_version", PG_SERVER_VERSION);
-        Messages.sendParameterStatus(channel, "server_encoding", "UTF8");
-        Messages.sendParameterStatus(channel, "client_encoding", "UTF8");
-        Messages.sendParameterStatus(channel, "datestyle", sessionSettings.dateStyle());
-        Messages.sendParameterStatus(channel, "TimeZone", "UTC");
-        Messages.sendParameterStatus(channel, "integer_datetimes", "on");
+    private void sendParams(Channel channel) {
+        for (String[] parameterStatus : PARAMETER_STATUS) {
+            Messages.sendParameterStatus(channel, parameterStatus[0], parameterStatus[1]);
```

Comment:
Could also leave this as is, and instead read all properties into a map in the test case and assert that.

Doesn't require visibility changes, and doesn't require new constants and can continue using the proper sessionSettings.